.github/resources/healthcheck.sh

```diff
@@ -0,0 +1,26 @@
+# Name of the target container to check
+container_name="$1"
```

Comment:
What do you think about making timeout as a second argument? That would allow us to have different timeout for different containers. Let me know if you think it will be useful at this point or not